bl/drawable/inc/drawable.hpp

```diff
@@ -29,14 +33,18 @@ class Drawable {
     const std::unordered_map<std::shared_ptr<Drawable>, Drawable::Position> &getChildDrawables()
         const;
     void addChild(const std::shared_ptr<Drawable> &child, Position p);
+    void removeChild(const std::shared_ptr<Drawable> &child);
```

Comment:
shouldn't be in this PR it is not fixing any clang-tidy issue

Reply:
fixed